linode_api4/groups/linode.py

```diff
@@ -284,6 +288,10 @@ def instance_create(
             )
             del kwargs["backup"]
 
+        if "firewall" in kwargs:
+            fw = kwargs["firewall"]
+            kwargs["firewall"] = fw.id if isinstance(fw, Firewall) else fw
+
```

Comment:
`firewall` field is now called `firewall_id` on the API endpoint. Should we make the change in SDK here or it's already handled somewhere else?

Reply:
Oops, looks like my mistake. I'll fix that real quick

Reply:
That was actually a recent update, originally it was `firewall`.